Makefile

```diff
@@ -6,11 +6,11 @@ ifndef $(GOPATH)
 endif
 # Build the default binary
 build: ui/package-lock.json
-	cd ui/ && ng build
```

Comment:
wouldn't `ui/package-lock.json` take care of this?

Reply:
it won't for first time installed, closing the comment as discussed offline